novawallet/Modules/AssetsSearch/AssetOperation/Swaps/SwapAssetsOperationInteractor.swift

```diff
@@ -13,7 +13,7 @@ final class SwapAssetsOperationInteractor: AnyCancellableCleaning {
     let settingsManager: SettingsManagerProtocol
 
     private let operationQueue: OperationQueue
-    private var builder: AssetSearchBuilder?
+    private var builder: SendAssetSearchBuilder?
```

Comment:
It is also model named as send passed to swaps. Let's also rename it to SpendAssetSearchBuilder